### DISCUSSION TITLE
Economic Output by NAICS-6 Codes

### DISCUSSION THREAD

**Discussion by smcnichols94** [USEPA/supply-chain-factors#62] [Q&A]
Hello,

In mapping the previous BEA codes to associated NAICs codes and factors, I have found that 13 BEA codes map to multiple unique emissions factors, since one BEA code most often maps to many different NAICs codes. I am wondering if there is a NAICS economic output model that could be used to take a weighted average to get a single emissions factor per BEA code where there is more than one unique factor? Or is there another approach to get at this? 

Thanks,
Susan

**Comment by bl-young** [comment]
That's correct, in some cases the mapping of BEA to NAICS can be many:many (e.g., in the construction sectors 23*). In the current release, factors by detailed BEA code are mapped to NAICS and weighted based on [commodity output](https://github.com/USEPA/supply-chain-factors/blob/main/R/Functions.R#L56-L65).

There is no data, to my knowledge, of output by NAICS.

To get a factor specific to the BEA codes, I would recommend running the R code and skipping the [step of mapping to NAICS](https://github.com/USEPA/supply-chain-factors/blob/main/CalculateEmissionFactors.Rmd#L67-L68), rather than trying to back them back out, in order to avoid the challenge you describe here for these specific cases.

**Reply by smcnichols94** [reply]
Thanks for the feedback. If I understand correctly, this means the emissions factors are derived by detailed BEA codes and then a weighted average of the detailed BEA codes that map to each NAICs codes is taken to get the factors by NAICs codes. Can you confirm that is correct? If so, is there a reason why 2 versions of the factors can't be provided in the emissions factor file, one by NAICs codes and one by detailed BEA codes?
